aws/resource_aws_wafv2_web_acl.go

```diff
@@ -824,9 +832,10 @@ func flattenWafv2ManagedRuleGroupStatement(r *wafv2.ManagedRuleGroupStatement) i
 	}
 
 	m := map[string]interface{}{
-		"excluded_rule": flattenWafv2ExcludedRules(r.ExcludedRules),
-		"name":          aws.StringValue(r.Name),
-		"vendor_name":   aws.StringValue(r.VendorName),
+		"excluded_rule":        flattenWafv2ExcludedRules(r.ExcludedRules),
+		"name":                 aws.StringValue(r.Name),
+		"vendor_name":          aws.StringValue(r.VendorName),
+		"scope_down_statement": nil,
 	}
 
```

Comment:
@Adirael I was able to replicate the error with the acceptance tests locally. Adding the following line to the `flattenWafv2ManagedRuleGroupStatement` seems to fix the issue. 
```suggestion
if r.ScopeDownStatement != nil {
		m["scope_down_statement"] = []interface{}{flattenWafv2Statement(r.ScopeDownStatement)}
}
```

Reply:
Thanks!! I'll try it out right now and incorporate it!